python/ray/experimental/locations.py

```diff
@@ -39,3 +39,30 @@ def get_object_locations(
     return ray._private.worker.global_worker.core_worker.get_object_locations(
         obj_refs, timeout_ms
     )
+
+
+def try_get_object_location_from_local(object_ref) -> Dict[str, Any] | None:
```

Comment:
For consistency with the other `get_object_locations` function, should we make this function take a list of objects?

Reply:
also, `get_local_object_locations` looks more concise

Reply:
done